scripts/copy_schemas.py

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env python3
+import requests
```

Comment:
what is wrong with curl here?

```
#!/usr/bin/env bash
set -e
curl https://raw.githubusercontent.com/mrc-ide/hintr/master/inst/schema/ModelRunOptions.schema.json > inst/schema/ModelRunOptions.schema.json
```

Reply:
(also this is why I like scripts with no extension, as you're free to change the implementation without updating README files)